tools/downloader/list_topologies.yml

```diff
@@ -3629,3 +3629,283 @@ topologies:
       - --output=prob
       - --input_model=$conv_dir/resnet-v1-50.onnx
     license: https://raw.githubusercontent.com/pytorch/vision/master/LICENSE
+
+  - name: "octave-densenet-121-0.125"
+    description: >-
+      The `octave-densenet-121-0.125` model is a modification of `densenet-121` from
+      the this paper <https://arxiv.org/pdf/1608.06993> with octave convolutions from
+      Drop an Octave: Reducing Spatial Redundancy in Convolutional Neural Networks
+      with Octave Convolution <https://arxiv.org/abs/1904.05049> with &alpha;=0.125.
```

Comment:
This is supposed to be plain text, so `&alpha;` isn't acceptable. You can either replace it with `alpha`, or with the Unicode alpha character.